src_assets/common/assets/web/config.html

```diff
@@ -705,6 +705,14 @@ <h2 class="accordion-header">
           </select>
           <div class="form-text">{{ $t('config.capture_desc') }}</div>
         </div>
+        <div class="mb-3" v-if="platform === 'windows'">
+          <label for="capture" class="form-label">{{ $t('config.capture') }}</label>
+          <select id="capture" class="form-select" v-model="config.capture">
+            <option value="ddx">Desktop Duplication API</option>
+            <option value="wgc">Windows.Graphics.Capture (BETA)</option>
+          </select>
+          <div class="form-text">{{ $t('config.capture_desc') }}</div>
+        </div>
```

Comment:
I added this to en_GB and en_US as well, for completeness' sake.

Are you looking for localizers?

Reply:
Please, only the `en.json`, our crowdin integration will ignore the others and it will cause merge conflicts.

> Are you looking for localizers?

Yes, for any languages other than en & en_US. Localization happens on crowdin at https://translate.lizardbyte.dev/